mne/io/nirx/nirx.py

```diff
@@ -21,15 +21,23 @@
 
 
 @fill_doc
-def read_raw_nirx(fname, preload=False, verbose=None):
+def read_raw_nirx(fname, saturated='ignore', preload=False, verbose=None):
     """Reader for a NIRX fNIRS recording.
 
-    This function has only been tested with NIRScout devices.
-
     Parameters
     ----------
     fname : str
         Path to the NIRX data folder or header file.
+    saturated : str
+        Replace saturated segments of data with NaNs.
+        If ignore (default) the measured data is returned, even if it
+        contains measurements while the amplifier was saturated.
+        If nan the returned data will contain NaNs during time segments
+        when the amplifier was saturated.
+        If annotate the returned data will contain annotations specifying
```

Comment:
```suggestion
        If "annotate" the returned data will contain annotations specifying
```
And I would say this should be the default actually